app/helpers/reactjs_helper.rb

```diff
@@ -11,58 +10,60 @@ def react_component(name, props = {})
     content_tag('foreman-react-component', '', :name => name, :data => { props: props })
   end
 
-  def webpacked_plugins_with_global_css
-    global_css_tags(global_plugins_list).join.html_safe
+  def js_tags_for(requested_plugins)
+    requested_plugins.map do |plugin|
+      javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{plugin.to_s.tr('-', '_')}','#{plugin.to_s.tr('-', '_')}','./index');".html_safe)
+    end
   end
 
   def webpacked_plugins_js_for(*plugin_names)
     js_tags_for(select_requested_plugins(plugin_names)).join.html_safe
   end
 
-  def webpacked_plugins_with_global_js
-    global_js_tags(global_plugins_list).join.html_safe
+  def other_webpack_plugin(plugin_name, file)
+    javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{plugin_name.to_s.tr('-', '_')}','#{plugin_name.to_s.tr('-', '_')}','./#{file}_index');".html_safe)
```

Comment:
Given the repetition of this conversion I think we should have a common way for it. I'm not exactly sure where that should live, but the Foreman plugin code feels like a natural place. Perhaps we already have such a thing.